addons/auth_totp/models/res_users.py

```diff
@@ -79,6 +79,61 @@ def _totp_try_setting(self, secret, code):
         _logger.info("2FA enable: SUCCESS for %s %r", self, self.login)
         return True
 
+    def get_totp_invite_url(self, user):
```

Comment:
Would make it a multi ensure_one method (having therefore ``if self.has_group('base.group_user')`` in override), allowing to call it directly from template